{"name": "test_criar_um_usu치rio_com_credenciais_v치lidas", "status": "broken", "statusDetails": {"message": "AttributeError: module 'pytest' has no attribute 'rng'", "trace": "fixturefunc = <function generate_user_data at 0x107880360>, request = <FixtureRequest for <Function test_criar_um_usu치rio_com_credenciais_v치lidas>>, kwargs = {}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.venv/lib/python3.13/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @given(\"que eu tenho um nome de usu치rio e uma senha v치lidos\")\n    def generate_user_data():\n        \"\"\"Gera credenciais v치lidas para o usu치rio\"\"\"\n        logging.info(\"游릭 [IN칈CIO] Gerando Usu치rio para Teste\")\n    \n        pytest.user_data = {\n>           \"userName\": \"usuario_teste_\" + str(pytest.rng.randint(1000, 9999)),\n            \"password\": \"Teste@123\"\n        }\nE       AttributeError: module 'pytest' has no attribute 'rng'\n\ntests/api/test_create_user.py:23: AttributeError"}, "description": "/Users/daniloamedeiros/Desktop/project-challenge-qa-acc/features/api_create_user.feature: Criar um usu치rio com credenciais v치lidas", "attachments": [{"name": "log", "source": "cc436f91-4f38-4d0d-89fb-478c817863e4-attachment.txt", "type": "text/plain"}], "start": 1739370867393, "stop": 1739370867395, "uuid": "32bdaf4d-c363-4dd3-b2da-ee2c3ab246ca", "historyId": "a7574963e73b147341298284211deff8", "testCaseId": "a7574963e73b147341298284211deff8", "fullName": "tests.api.test_create_user#test_criar_um_usu치rio_com_credenciais_v치lidas", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_create_user"}, {"name": "host", "value": "MacBook-Pro-de-Danilo.local"}, {"name": "thread", "value": "51217-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_create_user"}]}